test/integration/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Integration test code is using browser-specific features and special functions `output` and `import` that are used in different files. This is not the case in the rest of the code.